File: Threetexing/src/com/xinxin/Student.java
package com.xinxin;

public class Student extends Person{
	//private String name;
   // private int age;
    private String schoolName;
    
    public Student () {}
    public Student(String name,int age) {
    	//this.name=name;
    	//this.age=age;
    	this.setAge(age);
    	this.setName(name);
    }
    /*
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public int getAge() {
		return age;
	}
	public void setAge(int age) {
		this.age = age;
	}
    
	public void eat() {
		System.out.println("吃饭");
	}
	public void sleep() {
		System.out.println("睡觉");
	}
	*/
    
    public void info() {
    	eat();
    	//System.out.println("name:"+name);
    	System.out.println("我是一个学生");
    }


}

File: 类的属性和方法/类的属性和方法/student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace 类的属性和方法
{
    class student
    {
        //私有的，外面看不见，相当于封装
        private String name;
        private int age;

        //公共的，外面可以看见，相当于方法
        public void SetName(String name)
        {
            this.name = name;//让变量等于形式参数
        }

        public String getName( )
        {
            return name;
        }

        public void SetAge(int age)
        {
            this.age = age;
        }
        public int getAge()
        {
            return age;
        }

        public String getInfo()
        {
            return name + " " + age;
        }
    }
}
